ups-connector-product/README.md

```diff
@@ -1,74 +1,73 @@
 # UPS Modules
 
 ## Description
-This is a UPS modules made in Axon Ivy framework. It allows users to quickly and easily integrate with UPS Service. With this App, users can effortlessly manage their packages from UPS.
-With this connector:
-- Gives you full power to the OpenAPI UPS services.
-- Supports you with an easy-to-copy demo implementation to reduce your integration effort.
-
-UPS modules contains:
-- ups-connector
-- ups-oauth-feature
-- ups-connector-demo
-- ups-connector-product
-- ups-connector-test
-
-Common use cases for UPS APIs:
-- Shipping and tracking packages for your business
-- Obtaining shipping rates and time in transit for ecommerce
-- Creating visibility applications for your customers on your platform
-
-View the [API catalog](https://developer.ups.com/catalog) to identify which APIs meet your business needs.
-We hope you enjoy and we look forward to your contributions!
-
-## Set up
-
-1. Go to https://developer.ups.com, login or create a new UPS account. 
-2. Create an Application on UPS 
-3. After creating your application, the **Client ID** and **Client Secret** are generated and can be used to obtain an access token to authorize your API requests
-4. Configure the following variables in your project: 
-![Alt text](image-11.png)
+The Axon Ivy' [UPS connector](https://developer.ups.com/catalog) enables users to integrate UPS services seamlessly into any business process. This connector:
```

Comment:
"The Axon Ivy" and not "The Axon Ivy' "

Reply:
can you make this change and then you can comit and merge it ok?

Reply:
Thank you so much Andreas, have a nice day!